Fabric.Identity.API/Services/UserProfileService.cs

```diff
@@ -54,7 +60,16 @@ public async Task IsActiveAsync(IsActiveContext context)
             var sub = context.Subject.GetSubjectId();
             _logger.Debug($"found sub from IsActiveContext: {sub}");
             var user = await _userStore.FindBySubjectIdAsync(sub);
+
             context.IsActive = user != null;
+
+            if (user != null && _appConfig.AzureAuthenticationEnabled)

```

Comment:
Do we also have to check the scheme that the user logged in with? If we have Windows auth enabled and Azure auth enabled, we might set the user to inactive if they are a windows user and we don't have an issuer for them.

Reply:
good point.  I wonder how this is working!

Reply:
updated